### ISSUE TITLE
Cannot launch my project

### ISSUE THREAD

**Issue by Flymeth** [microsoft/vscode-edge-devtools#873]
**Environment (please complete the following information):**
 - OS (*e.g.,* Windows 10): win11
 - Extension version (*e.g.,* 1.2.3): 1.4.3

**Describe the bug:**

Nothing append when I launch my project with the extention

**Repro steps:**

Go on the edge icon on the navbar, then create the launch.json file then click on "Launch project": nothing append (or something try to start but it cannot)

**Expected behavior:**

I expected that the extention works ?

**Comment by rrikon** [comment]
I have the same problem as you.

OS: Windows10
Extension version: v1.4.3

**Comment by geoffrey-c** [comment]
same here

os: macOS 11.6.3

**Comment by chris-miner** [comment]
> Create a directory/folder with the name of `vscode-edge-devtools-userdatadir_9222` inside `C:\Users\%username%\AppData\Local\Temp\` for now while we wait for the merge to be released as an update.

What would the fix be on macOS?

**Comment by mserwicki** [comment]
> What would the temporary work-around be on macOS?

I was able to determinate my temp directory on mac by running from terminal

```
node -pe 'os.tmpdir()'
```

then cd into it, created the `vscode-edge-devtools-userdatadir_9222` there, and it started to work

**Comment by chris-miner** [comment]
> then created the `vscode-edge-devtools-userdatadir_9222` there, and it started to work
This got me a little further.  Thanks!  Now the "launch instance" button does something.  It opens two tabs — Edge DevTools and Edge DevTools: Screencast.  The first tab reports "Unable to download DevTools for the current target", the second tab shows me the contents of "file:///Users/username/.vscode/extensions/ms-edgedevtools.vscode-edge-devtools-1.4.3/out/startpage/index.html"

I generated a launch.json file, edited the url entry, and this got me to my project page, but still with no dev tools loading.

**Comment by hkal** [comment]
Hey there - y'all are running into two issues that we're working to resolve:

1. **User data-dir not found** - we recently merged a fix for this (#871). Turns out the issue is more widespread than anticipated, so we'll be pushing an updated package with the fix today.
2.  **"Unable to download DevTools for the current target"** - the extension downloads devtools UI that matches the target browser from a CDN. We're experiencing issues where some packages are not being published and are looking into the cause now. Using stable `98.0.1108.50 (Official build) (64-bit)` should unblock you. It's downloadable [here](https://www.microsoft.com/en-us/edge).

@mserwicki - What version of Edge are you using? You can find out by navigating to `edge://version/`. Knowing this will help us debug.

**Comment by mserwicki** [comment]
> Hey there - y'all are running into two issues that we're working to resolve:
> 
> 1. **User data-dir not found** - we recently merged a fix for this ([Bug Fix: create userDataDir if it doesn't exist #871](https://github.com/microsoft/vscode-edge-devtools/pull/871)). Turns out the issue is more widespread than anticipated, so we'll be pushing an updated package with the fix today.
> 2. **"Unable to download DevTools for the current target"** - the extension downloads devtools UI that matches the target browser from a CDN. We're experiencing issues where some packages are not being published and are looking into the cause now. Using stable `98.0.1108.50 (Official build) (64-bit)` should unblock you. It's downloadable [here](https://www.microsoft.com/en-us/edge).
> 
> @mserwicki - What version of Edge are you using? You can find out by navigating to `edge://version/`. Knowing this will help us debug.

```
Microsoft Edge | 98.0.1108.51 (Official build) (x86_64)
-- | --
Revision | 2ca6a1220d097f76c0cdae771305c0e2a72c8bf6
Operating system | macOS Version 12.2.1 (Build 21D62)
JavaScript | V8 9.8.118.4
User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.82 Safari/537.36 Edg/98.0.1108.51
```

**Comment by chris-miner** [comment]
> 2\. Using stable `98.0.1108.50 (Official build) (64-bit)` should unblock you. It's downloadable [here](https://www.microsoft.com/en-us/edge).

Thanks for the info.  Turns out I'm using `98.0.1108.51 (Official build) (x86_64)`.  Looks like the download is pointed at `MicrosoftEdge-98.0.1108.51` for now which is what I have installed.  So that doesn't seem like it will unblock me. :(

**Comment by hkal** [comment]
#886 should address the problem. We'll publish a new update when it's merged.

**Comment by chris-miner** [comment]
This is now working for me.  I'm on Edge Browser version `98.0.1108.55` and version `1.4.4` of edge tools.

**Comment by hkal** [comment]
This issue should be fixed in 1.4.5 (#888). Re-open if that is not the case.
